pytest.ini

```diff
@@ -58,8 +58,10 @@ filterwarnings =
     # The following are raised by the py310-oldestdeps job
     ignore:distutils Version classes are deprecated
     ignore:ERFA function *
-    # This seems to randomly crop up but I don't know why
+    # This seems to randomly crop up due to our network stack
+    # One day we need to track down the root cause
```

Comment:
does this comment refer to just your new ignore or all of these?

Reply:
All of these.